i18n/appointments/locale_en.json

```diff
@@ -244,5 +243,8 @@
   "CHANGE_DATE_TO_LABEL": "Change ${moment(originalAppointmentDate).format('Do MMM')} to",
   "PATIENT_DOUBLE_BOOKING_CONFLICT_MESSAGE": "Current {currentAppointmentService} request conflicts with {existingAppointmentService} appointment on and",
   "NO_SERVICE_CONFLICTS_DEFAULT_TEXT": "The {label} service you had selected for the appointment(s) is not available during below listed dates",
-  "PROVIDER_ERROR_MESSAGE": "Please select only a maximum of {maxAppointmentProviders} provider(s)"
+  "PROVIDER_ERROR_MESSAGE": "Please select only a maximum of {maxAppointmentProviders} provider(s)",
+  "APPOINTMENT_BOOKING_MESSAGE": "Appointment for #patientName, #patientId on #dateTime for #service is booked at #clinicName. For any queries call us on +919876543210.",
+  "RECURRING_APPOINTMENT_BOOKING_MESSAGE": "A recurring appointment for #patientName, #patientId for #service is booked for every #frequency at #clinicName. Your next appointment is at #dateTime. For any queries call us on +919876543210.",
+  "TELECONSULTATION_APPOINTMENT_BOOKING_MESSAGE": "A teleconsultation appointment for #patientName, #patientId on #dateTime for #service is booked at #clinicName. Please click on the link #meetingLink on #dateTime to join the consultation. For any queries call us on +919876543210."
```

Comment:
The phone number seems realistic. Whose number is this? 
I would prefer to make this a configuration which can be edited via some UI, and not require a redeployment of clinic config.

Reply:
it's realistic but not belongs to anyone. 9876543210 (reverse of 0123456789)

Reply:
Refactored to read phoneNumber from global property so that it can be updated when needed